### PR TITLE
Update protobuf to 3.19.1

### PR DESCRIPTION
**Pull request by thii**

### CODE REVIEW

**Comment by comius** [comment]
I've mirrored the protoc file. Please update the code. Thanks!

**Review by comius — CHANGES_REQUESTED** [review]
I've uploaded the file. Please resolve the TODO. Thanks!

**Comment by thii** [comment]
Updated. Thanks!

File: proto/private/dependencies.bzl [hunk 1]
```diff
@@ -23,11 +23,11 @@ dependencies = {
...
+        "sha256": "87407cd28e7a9c95d9f61a098a53cf031109d451a7763e7dd1253abf8b4df422",
+        "strip_prefix": "protobuf-3.19.1",
         "urls": [
-            "https://mirror.bazel.build/github.com/protocolbuffers/protobuf/archive/v3.17.0.tar.gz",
-            "https://github.com/protocolbuffers/protobuf/archive/v3.17.0.tar.gz",
+            "https://mirror.bazel.build/github.com/protocolbuffers/protobuf/archive/v3.19.1.tar.gz",
+            "https://github.com/protocolbuffers/protobuf/archive/v3.19.1.tar.gz",
         ],
     },
     "com_google_protobuf_protoc_linux_aarch64": {
```
Comment (Yannic) [inline proto/private/dependencies.bzl:33]: Can you also update the pre-compiled files below?

**Review by Yannic — CHANGES_REQUESTED** [review]
Thanks for taking this on!

File: proto/private/dependencies.bzl (hunk 1 above)
Comment (thii) [reply proto/private/dependencies.bzl:33]: Updated. Can you help create mirrors for those files?

Comment (comius) [reply proto/private/dependencies.bzl:33]: On it.

**Review by comius — APPROVED** [review]

**Review by Yannic — APPROVED** [review]

**Pull request merged**